src/org/usfirst/frc/team2585/systems/ShooterSystem.java

```diff
@@ -94,8 +88,8 @@ public void run() {
 	 */
 	@Override
 	public void destroy() {
-		agitator = null;
-		shooter = null;
+		((PWM) agitator).free();
```

Comment:
This is technically correct, but you should always use check to see if an object is an instance of a class before typecasting it. Use if (agitator instance PWM).